themes/default/views/editor/objects/screen_html.php

```diff
@@ -82,4 +82,56 @@
 
 	<div class="editorBottomPadding"><!-- empty --></div>
 	
-	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
\ No newline at end of file
+	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
+	<script type="text/javascript">
+		(function(initSplitSuffix) {
+
+			initSplitSuffix(window.jQuery);
+
+		}(function($) {
+
+				$(function () {
+					var reg = /^\s*\.?\s?(\d*)\s*(.*)\s*$/,
+						parse = function (val) {
+							if (val && typeof val === 'string') {
```

Comment:
This function is returning an array sometimes, and `undefined` other times.  That's fine but it should be explicit, so at minimum just add `return undefined` at the end of the function.  

However (and this is purely a style thing) if you also invert the condition here, you can explicitly return the `undefined` there and unindent the main part of the function, which I find neat:

````
if (!val || typeof val !== string) {
    return undefined;
}
var parts = ...
````